cmd/util/ledger/migrations/atree_register_migration.go

```diff
@@ -144,9 +161,60 @@ func (m *AtreeRegisterMigrator) MigrateAccount(
 		})
 	}
 
+	// Check storage health after migration, if enabled.
+	if m.checkStorageHealthAfterMigration {
```

Comment:
Maybe only do this iff the health was not checked before, or if it was checked before and it was healthy.

For example, we already do that in the Cadence 1.0 migration: https://github.com/onflow/flow-go/blob/c58eb78a2585f70fbb8daf053d96e8b0597e74a1/cmd/util/ledger/migrations/cadence_values_migration.go#L177-L180

Reply:
Actually, atree migration currently filters out unreachable payloads so storage health check after migration will pass even if health check before migration fails.  

After we add new migration step to handle unreachable payloads, I can update health check here for atree migration.

Reply:
Ah, nice!